src/common.css

```diff
@@ -190,7 +194,8 @@ body > .select2-container.open {
 }
 
 /* See Scrollable Menu with Bootstrap 3 http://stackoverflow.com/questions/19227496 */
-.ui-select-bootstrap > .ui-select-choices ,.ui-select-bootstrap > .ui-select-no-choice {
+.ui-select-container > .ui-select-choices-bootstrap,
```

Comment:
The select2 and selectize themes don't do styling with the same sort of classes - as far as I can tell, the css for those shouldn't be affected by this. I did some testing with the other themes as well, and they seemed to work with append-dropdown-to-body!